swap/peer.go

```diff
@@ -109,7 +108,7 @@ func (p *Peer) updateBalance(amount int64) error {
 	if err := p.setBalance(newBalance); err != nil {
 		return err
 	}
-	log.Debug("balance for peer after accounting", "peer", p.ID().String(), "balance", strconv.FormatInt(newBalance, 10))
+	p.swap.logger.Debug("balance for peer after accounting", "peer", p.ID().String(), "balance", strconv.FormatInt(newBalance, 10))
```

Comment:
This would be too long, we should strive to get it shorter, maybe `slog`, or something nicer.

Reply:
This is the kind of stuff I'd want to see in the swap logger. Everything related to financial values, an audit trail.